helm/charts/oathkeeper/README.md

```diff
@@ -8,113 +8,114 @@ A Helm chart for deploying ORY Oathkeeper in Kubernetes
 
 ## Maintainers
```

Comment:
Those files are auto-generated by helm-docs plugin from our values.yaml. The `make format` and the plugin will override each other all the time 😞